frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -128,12 +132,16 @@ class Chart extends Component<Props, State> {
     });
   }
 
+  private checkIfMobile = () => {
+    this.setState({ isMobile: window.innerWidth <= 640 });
+  };
+
```

Comment:
`640`px as this is the breakpoint that the page uses when the layout starts changing for "mobile". 

This fn is used to easily determine the screen size. This way we can dynamically change the chart's option without having to initialise/create a second chart only for mobile.